### PR TITLE
Improved MultiReadings

### PR DESCRIPTION
**Pull request by darrylmasson**
By "improved" I mean that it should now work more or less as intended. The db will need a minor update before deployment.

### CODE REVIEW

File: Doberman/SensorMonitor.py [hunk 1]
```diff
@@ -17,12 +17,12 @@ def setup(self):
         for rd in cfg_doc['readings'].keys():
             self.logger.debug('Constructing ' + rd)
             reading_doc = self.db.get_reading_setting(self.name, rd)
-            kwargs = {'sensor_name': self.name, 'reading_name': rd, 'logger': self.logger,
-                      'event': self.event, 'db': self.db, 'sensor': self.sensor}
-            if 'is_multi' in reading_doc:
+            kwargs = {'reading_name': rd, 'logger': self.logger, 'db': self.db,
+                      'event': self.event, 'sensor': self.sensor}
+            if 'is_multi' in reading_doc and isinstance(reading_doc['is_multi'], list):
```
Comment (jarongrigat) [inline Doberman/SensorMonitor.py:22]: Ok, so it should be `is_multi` then.

File: Doberman/SensorMonitor.py [hunk 2]
```diff
@@ -99,4 +99,4 @@ def reload_readings(self):
             self.readings[reading_name] = Doberman.Reading(self.name, reading_name,
                                                            self.db)
             self.register(func=self.ScheduleReading, period=r.readout_interval,
```
Comment (jarongrigat) [inline Doberman/SensorMonitor.py:101]: Unrelated, but I don't think this line works. What is `r`?

**Comment by darrylmasson** [comment]
I decided `multi_reading` was the clearest solution.

File: Doberman/SensorMonitor.py (hunk 2 above)
Comment (darrylmasson) [reply Doberman/SensorMonitor.py]: I thought I had already rewritten that function but that was probably lost in the Great Push

**Review by jarongrigat — APPROVED** [review]

**Pull request merged**